hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaTargetFilterQuery.java

```diff
@@ -65,7 +65,8 @@
     @Column(name = "auto_assign_action_type", nullable = true)
     @ObjectTypeConverter(name = "autoAssignActionType", objectType = Action.ActionType.class, dataType = Integer.class, conversionValues = {
             @ConversionValue(objectValue = "FORCED", dataValue = "0"),
-            @ConversionValue(objectValue = "SOFT", dataValue = "1") })
+            @ConversionValue(objectValue = "SOFT", dataValue = "1"),
+            @ConversionValue(objectValue = "DOWNLOAD_ONLY", dataValue = "2")})
```

Comment:
TimeForced is missing which was translating to value '2', and DOWNLOAD_ONLY was translated to value '3' somewhere else in the code. What to do?

Reply:
reuse the same converter from JpaAction